### PR TITLE
Iotupd 2326: Add support for BLOCK2 blockwise messages GET req handling and fix bug

### PR DESCRIPTION
**Pull request by Jekaah**

### CODE REVIEW

**Comment by Jekaah** [comment]
Basic changes added to get BLOCK2 blockwise messages working. Depends on https://github.com/ARMmbed/mbed-cloud-client-internal/pull/390 and https://github.com/ARMmbed/mbed-client-testapp/pull/644

File: source/sn_coap_protocol.c [hunk 1]
```diff
@@ -1902,6 +1889,34 @@ static sn_coap_hdr_s *sn_coap_handle_blockwise_message(struct coap_s *handle, sn
...
+                            tr_error("sn_coap_handle_blockwise_message - failed to allocate for uri path ptr!");
+                            return NULL;
+                        }
+                        memcpy(src_coap_blockwise_ack_msg_ptr->uri_path_ptr, previous_blockwise_msg_ptr->coap_msg_ptr->uri_path_ptr, previous_blockwise_msg_ptr->coap_msg_ptr->uri_path_len);
+                    }
+                }
+
+                ns_list_remove(&handle->linked_list_blockwise_sent_msgs, previous_blockwise_msg_ptr);
+                if( previous_blockwise_msg_ptr->coap_msg_ptr ){
+                    if(previous_blockwise_msg_ptr->coap_msg_ptr->payload_ptr){
```
Comment (TeroJaasko) [inline source/sn_coap_protocol.c]: Please fix the code style; ("if<space>(..)<space>{" here too, just as you did in lines 1894, 1896, etc.

**Comment by Jekaah** [comment]
rebuild

**Comment by Jekaah** [comment]
rebuild

**Comment by yogpan01** [comment]
@anttiylitokola Please check this one.

**Comment by Jekaah** [comment]
rebuild

**Pull request merged**